packages/components/dialog/src/dialog.scss

```diff
@@ -126,18 +109,18 @@ slot[name='footer'] {
 }
 
 dialog[closing] {
-  animation: slide-down var(--_closing-duration) ease-out;
+  animation: slide-down var(--_closing-duration) both ease-out;
 
   &::backdrop {
```

Comment:
It looks like the backdrop is not animated in Firefox?

Reply:
As discussed in Slack, I see no way to fix this in FF. I can't even see the `::backdrop` pseudo element in the FF devtools.